content/guides/authentication.md

```diff
@@ -29,39 +41,27 @@ curl https://auth.dfuse.io/v1/auth/issue -s --data-binary '{"api_key":"web_abcde
 {{< /tab >}}
 {{< /tabs >}}
 
-If you haven't already, head out to our [self-service API management portal](https://app.dfuse.io), sign up and create a long-term API key.
-
-Once you have this API key, call the [`https://auth.dfuse.io/v1/auth/issue` endpoint](#post-v1-auth-issue) to get a fresh Authentication Token (JWT).
-
-By default, each JWT is **valid for a period of 24 hours**. Make sure you cache those values to avoid hitting rate limiting on JWT issuance.
+Our `client-JS` library will handle the token issuance and caching for you if you decide to utilize it. You can also consult a <a href="https://github.com/dfuse-io/client-js/blob/master/examples/reference/auth-issue.ts" title="JWT Issuance Example">working reference example</a> if you would like to follow along with some sample TypeScript code.
 
 {{< tip >}}
-See <https://jwt.io/introduction/> to learn more about JWTs.
+To learn more about JWTs, you can consult <https://jwt.io/introduction/>.
 {{< /tip >}}
 
+Each JWT is **valid for a period of 24 hours**. Make sure to cache those values to avoid hitting rate limiting on JWT issuance. A connection will be validated upon a valid connection being made and will not need to be reauthenticated until a new connection needs to be reestablished. 
+
 ## Lifecycle of Short-Lived JWTs
 
-Each JWT has an expiration date, so it is important to take that into account before making a request to the _dfuse_ APIs. Here is the recommended procedure to take before making a request:
+Each JWT has an expiration date, so it is important to take that into account before making a request to the dfuse APIs. Here is the recommended procedure to take before making a request:
 
 1. Retrieve the JWT, and examine it's expiration time
-1. If the JWT is past it's expiration time, is near expiration or is absent from cache, fetch a new one through the `/v1/auth/issue` endpoint, and cache the response.
-1. Call _dfuse_ with the valid JWT token.
+1. If the JWT is past its expiration time, is near expiration, or is absent from cache, fetch a new one through the `/v1/auth/issue` endpoint, and cache the response.
```

Comment:
Should we make that link to the endpoint's page in reference?

Reply:
Still needs to be figured out and fixed.